pkg/resource/logging-config/alloy/logging.alloy.template

```diff
@@ -228,10 +242,22 @@ loki.process "kubernetes_audit" {
 		source = "objectRef"
 	}
 
+	stage.structured_metadata {
+		values = {
+			"resource" = "",
+			"filename" = "",
+		}
+	}
+
+	stage.label_drop {
+		values = [
+			"filename",
+		]
+	}
+
 	stage.labels {
 		values = {
-			namespace = null,
-			resource  = null,
+			namespace = "",
```

Comment:
What is the difference between `namespace = null` and `namespace = ""`?

Reply:
I'm not sure but in doubt, I followed the Doc https://grafana.com/docs/alloy/latest/reference/components/loki/loki.process/#stagelabels-block